data/brands/shop/variety_store.json

```diff
@@ -517,6 +517,19 @@
         "shop": "variety_store"
       }
     },
+    {
+      "displayName": "Pasar Ekonomi Econsave",
+      "id": "pasarekonomieconsave-c7d002",
+      "locationSet": {"include": ["my"]},
+      "tags": {
+        "brand": "Pasar Ekonomi Econsave",
+        "brand:zh": "宜康省经济市场",
+        "brand:wikidata": "Q61776108",
```

Comment:
https://www.wikidata.org/wiki/Q61776108 looks wrong - it goes to something unrelated?